pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -1314,6 +1314,8 @@ const (
 	FilterSpecExternalID = "spec.externalID"
 	// SpecClusterServiceClassName is only used for plans, the parent service class name.
 	FilterSpecClusterServiceClassName = "spec.clusterServiceClass.name"
+	// SpecClusterServiceBroker is used for plans to filter by broker
+	FilterSpecClusterServiceBrokerName = "spec.clusterServiceBroker.name"
```

Comment:
I assumed wrong, it looks like spec has the name, this property should be `servicePlan.spec.clusterServiceBrokerName` . <-- it is the json path of the object being referenced.

Reply:
One last detail. I think I need input from @jeremyrickard about the usage of "Cluster" in this property. I am debating about if it is a big deal to leak the fact that this predicate is applied to a cluster or namespaced scoped broker...

I am leaning towards we should eliminate `cluster` here in general and just call it `FilterSpecClusterServiceBrokerName = "spec.brokerName"` even if that is a lie for Cluster Scoped Brokers. 

I think this could be a rework later on when we add support for filtering on namespaced classes and plans.

Reply:
1.) Would the incorrect spec cause my build error that I mention in slack regarding defaulter-gen? My guess is that doesn't.

2.) I don't understand the leaking part and what implications it has? I dont know if this is the right place to explain or if its a slack conversation.